lib/joi/LICENSE

```diff
@@ -0,0 +1,173 @@
+===
```

Comment:
License texts copied verbatim from the repositories of Joi and each of its runtime dependencies.